my-app/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,31 @@
+import React,{ReactNode} from 'react';
+import "./style.css"
+
+interface IMenu{
+    menuClass: string;
+    children:ReactNode
+}
+const Menu = ({menuClass,children}: IMenu) => {
+    const handleBurgerClick = (event: any,b: number) =>{
```

Comment:
Remove any. Type of event should be something like `React.MouseEvent<HTMLElement>`.